### ISSUE TITLE
Problem in code execution :: epoch limit reached error

### ISSUE THREAD

**Issue by myaimmystyle** [johnsmithm/handwritten-tf-1.0#8]
I created tfrecords file and tried to run this below command.

python train.py --slices 26 --width 12 --stride 3 --Bwidth 90 --train_data_pattern batch/tf-data-40x320-81/handwritten-test-{}.tfrecords --train_dir separable_lstm --test_data_pattern batch/tf-data-40x320-81/handwritten-test-{}.tfrecords  --max_steps 6000 --batch_size 20 --beam_size 3 --input_chanels 1 --model MDLSTMCTCModel --base_learning_rate 0.001 --num_readers 2 --export_model_steps 500 --display_step 10 --display_step_lme 100 --start_new_model

***************************************************************************
I get this below output:: simply throws exception as epoch limit reached

name: GeForce 940MX
major: 5 minor: 0 memoryClockRate (GHz) 1.2415
pciBusID 0000:01:00.0
Total memory: 1.96GiB
Free memory: 1.59GiB
2018-01-04 18:22:40.984476: I tensorflow/core/common_runtime/gpu/gpu_device.cc:908] DMA: 0 
2018-01-04 18:22:40.984483: I tensorflow/core/common_runtime/gpu/gpu_device.cc:918] 0:   Y 
2018-01-04 18:22:40.984492: I tensorflow/core/common_runtime/gpu/gpu_device.cc:977] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce 940MX, pci bus id: 0000:01:00.0)
INFO:tensorflow:/job:master/task:0: Entering training loop.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:separable_lstm/model.ckpt-0 is not in all_model_checkpoint_paths. Manually adding it.
2018-01-04 18:22:42.531717: W tensorflow/core/kernels/queue_base.cc:302] _3_test_input/shuffle_batch_join/random_shuffle_queue: Skipping cancelled dequeue attempt with queue not closed
2018-01-04 18:22:42.532438: W tensorflow/core/kernels/queue_base.cc:302] _3_test_input/shuffle_batch_join/random_shuffle_queue: Skipping cancelled dequeue attempt with queue not closed
INFO:tensorflow:/job:master/task:0: Done training -- epoch limit reached.
INFO:tensorflow:/job:master/task:0: Exited training loop.
(py27_tensorflow_gpu1.1) mlx@mlx-icr:~/Tensorflow_codes/tensorflow_intellichars/handwritten-tf-1.0-master$ 

*************************************************************************************************
Please help

**Comment by iyvinjose** [comment]
I am also facing this issue..

**Comment by myaimmystyle** [comment]
My laptop has only 2 GB of Graphics memory. Is that an issue??

**Comment by johnsmithm** [comment]
2GB is not enough
